asf_lifetime_cost_model/config/base.yaml

```
@@ -0,0 +1,61 @@
+# List of possible arguments for a variety of parameters and default values
+#that enable the calculation of the lifetime cost of different heating systems.
+
+ashp_efficiency_options: [
+    "reference", # SCOP 3
+    "high", # SCOP > 3
+    "low", # SCOP < 3
+  ]
+
+boiler_efficiency_default: 0.85 # boiler efficiency
+
+life_span_default: # in years
+  ashp: 15 # years
+  boiler: 15 # years
+
+maintenance_costs_default:
+  ashp:
+    annual_cost: 150 # GBP
+    number_of_years: 5 # HP warranty
+  boiler:
+    annual_cost: 80 # GBP
+    number_of_years: 15 # boiler lifetime
+
+ashp_subsidy_options:
+  [
+    "flat",
+    "slow stepdown",
+    "fast stepdown",
+    "high",
+    "zero from 2028",
+    "smallest",
+    "no subsidy",
+  ]
+
+ashp_market_cost_change: [
+    { "name": "optimistic", "annual_price_reduction": 0.045 }, # 4.5% annual price reduction
+    { "name": "moderate", "annual_price_reduction": 0.025 }, # 2.5% annual price reduction
+    { "name": "conservative", "annual_price_reduction": 0.01 }, # 1% annual price reduction
+  ]
+
+ashp_loans_options: [True, False] # whether ASHPs are purchased with loans
+loan_interest_rate_default: 0.05 # percent
+
+wholesale_price_projection_options: [
+    "reference", # DESNZ_Ref
+    "low fossil fuel prices", # DESNZ_Low
+    "high fossil fuel prices", # DESNZ_High
+  ]
+
+levy_rebalancing_options: [
+    "reference", # no change to policy costs in current price cap
+    "remove all", # removes all from electricity and gas
+    "rebalance RO and FiT to gas", # RO and FiT moved to gas
+    "remove from electricity", # removed from electricity
+    "rebalance electricity unit cost levies to gas", # move all unit levies to gas
+    "Rebalance all to gas", # all levies moved to gas
```

```suggestion
    "rebalance all to gas", # all levies moved to gas
```

Hi @sofiapinto, just standardising these to be the same as the valid arguments for the get_rebalanced_levies getter

---

won't commit, since this all changed with the policy team input